packages/web-components/fast-foundation/src/carousel/carousel.spec.md

```diff
@@ -49,66 +51,76 @@ As defined by the W3C:
 - `aria-label` - optional, based on implementation**
 - `paused` - boolean whether the rotation is paused or not
 - `activeslideid` - string
+- `basicpattern` - boolean whether or not the carousel follows the basic or tabbed pattern per ARIA spec, defaults to tabbed when basicpattern is false
```

Comment:
is "basicpattern" terminology that we came up with? It doesn't seem very descriptive to me so if so I think we should look for a more descriptive name. If we're referencing a named pattern from ARIA or some other resource, can we link off to it to provide additional context?